action.yaml

```diff
@@ -22,7 +22,7 @@ inputs:
     requires: false
 runs:
   using: docker
-  image: docker://ghcr.io/xenitab/gitops-promotion:v0.1.0
+  image: docker://ghcr.io/xenitab/gitops-promotion:dev
   entrypoint: /usr/local/bin/action-entrypoint.sh
```

Comment:
Is this intentional?

Reply:
In order to actually test this PR, we need to update the `action.yaml` definition to point to a custom image. It goes something like this: gitops-repo/.github/workflows/new.yaml -> gitops-promotion/action.yaml@branch-name -> ghcr.io/image:custom-tag. The `dev` tag was built manually. Will need to remember not to merge this change.

Reply:
Tag update removed.